packages/e2e/features/ui/components/authenticator/sign-in-sms-mfa.feature

```diff
@@ -41,3 +41,17 @@ Feature: Sign In with SMS MFA
     And I type my password
     And I click the "Sign in" button
     Then I see "User does not exist"
+
+@angular @react @vue
+  Scenario: Sign in with force change password credentials with sms mfa
+    Given I intercept '{ "headers": { "X-Amz-Target": "AWSCognitoIdentityProviderService.RespondToAuthChallenge" } }' with fixture "force-change-password"
+    When I select my country code with status "FORCE_CHANGE_PASSWORD"
```

Comment:
I think even if the force change password credentials expire, as long as we intercept it here, we'll be OK